website/docs/r/fabric_capacity.html.markdown

```diff
@@ -0,0 +1,86 @@
+---
+subcategory: "Fabric"
+layout: "azurerm"
+page_title: "Azure Resource Manager: azurerm_fabric_capacity"
+description: |-
+  Manages a Fabric Capacity.
+---
+
+# azurerm_fabric_capacity
+
+Manages a Fabric Capacity.
+
+## Example Usage
+
+```hcl
+data "azurerm_client_config" "current" {}
+
+resource "azurerm_resource_group" "example" {
+  name     = "example-resources"
+  location = "West Europe"
+}
+
+resource "azurerm_fabric_capacity" "example" {
+  name                = "example-ffc"
+  resource_group_name = azurerm_resource_group.example.name
+  location            = "West Europe"
+
+  administration_members = [data.azurerm_client_config.current.object_id]
+
+  sku {
+    name = "F32"
+    tier = "Fabric"
+  }
+
+  tags = {
+    environment = "test"
+  }
+}
+```
+
+## Arguments Reference
+
+The following arguments are supported:
+
+* `name` - (Required) The name which should be used for the Fabric Capacity. Changing this forces a new resource to be created.
+
+* `location` - (Required) The supported Azure location where the Fabric Capacity exists. Changing this forces a new resource to be created.
+
+* `resource_group_name` - (Required) The name of the Resource Group in which to create the Fabric Capacity. Changing this forces a new resource to be created.
+
+* `sku` - (Required) A `sku` block as defined below.
+
+* `administration_members` - (Optional) An array of administrator user identities. The member must be a member user or a service principal.
```

Comment:
What is a member user?

Reply:
I assume that member user means the [user type](https://registry.terraform.io/providers/hashicorp/azuread/latest/docs/resources/user#user_type-1) is `Memeber` instead of `Guest`.